Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan.yml

```diff
@@ -1,3264 +1,2463 @@
+      "72_73_#default#": 0.76,
+      "75_48_#default#": 0.52,
+      "75_76_yes": 0.59,
+      "77_78_#default#": 0.54
+    },
+    "paper": {
+      "dimensions": {
+        "height": 4435,
+        "width": 3860,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: BlockAttackerIP
+  value:
+    simple: "False"
+  required: false
+  description: Whether attacking IPs should be automatically blocked using firewalls.
+  playbookInputQuery:
+- key: WhitelistedHostnames
+  value:
+    simple: "AdminPC"
+  required: false
+  description: A list of comma-separated hostnames that should not be isolated even if used in an attack.
+  playbookInputQuery:
+- key: EmailAddressesToNotify
+  value: {}
+  required: false
+  description: A list of comma-separated values of email addresses that should receive a notification about compromised hosts.
+  playbookInputQuery:
+- key: CriticalUsernames
+  value: {}
+  required: false
+  description: A list of comma-separated names of critical users in the organization. This will affect the calculated severity of the incident.
+  playbookInputQuery:
+- key: IsolateEndpointAutomatically
+  value:
+    simple: "False"
+  required: false
+  description: Whether to automatically isolate endpoints, or opt for manual user approval. True means isolation will be done automatically.
+  playbookInputQuery:
+- key: InternalIPRanges
+  value: {}
+  required: false
+  description: 'A list of IP ranges to check the IP against. The list should be provided in CIDR notation, separated by commas. An example of a list of ranges would be: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" (without quotes). If a list is not provided, will use default list provided in the IsIPInRanges script (the known IPv4 private address ranges).'
+  playbookInputQuery:
+- key: CriticalHostnames
+  value:
+    simple: AdminPC
+  required: false
+  description: 'A list of comma-separated names of critical endpoints in the organization. This will affect the calculated severity of the incident.'
+  playbookInputQuery:
+- key: RoleForEscalation
+  value: {}
+  required: false
+  description: The name of the Cortex XSOAR role of the users that the incident can be escalated to in case of developments like lateral movement.
+  playbookInputQuery:
+- key: BlockMaliciousFiles
+  value:
+    simple: "False"
+  required: false
+  description: Whether to automatically block malicious files involved with the incident across all endpoints in the organization.
+  playbookInputQuery:
+- key: CriticalADGroups
+  value:
+    simple: "admins"
+  required: false
+  description: CSV of DN names of critical Active Directory groups. This will affect the severity calculated for this incident.
+  playbookInputQuery:
+- key: OnCall
+  value:
+    simple: "false"
+  required: false
+  description: Set to true to assign only the users that are currently on shift.
+  playbookInputQuery:
+outputs: []
+fromversion: 5.0.0
+tests:
+- Test XDR Playbook
```

Comment:
```suggestion
- no test
```

Reply:
This test was incorrectly added by the SDK, let's revert it

Reply:
done